ui/components/src/DateTimeRangePicker/AbsoluteTimePicker.tsx

```diff
@@ -126,34 +135,38 @@ export const AbsoluteTimePicker = ({ initialTimeRange, onChange }: AbsoluteTimeF
             />
           </Box>
         )}
-        <TextField
-          fullWidth
-          onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
-            // TODO: add helperText, fix validation after we decide on form state solution
-            updateDateRange(event.target.value, true);
+        <Grid
+          container
+          spacing={1}
+          sx={{
+            maxWidth: 312,
```

Comment:
Can we use a stack instead of grid here? And maybe use a constant for max width?